.changeset/khaki-glasses-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/sdk-react": patch
+---
+
+Patch EWK custom session lengths and fix useLocalStorage issues
```

Comment:
Can we add a little more context here?

i.e. what were the local storage issues we were seeing, and what was the issue with custom session lengths? In other words are we fixing an existing bug/broken state, or adding a net new feature (such as being able to pass through expiration seconds)?